SpecFlowProject10/Features/Calculator.feature

```diff
@@ -1,19 +1,21 @@
 ﻿
 
 
+Feature: Peform Operation On Calculator
 
-Feature: Perform  Operations on  Calculator Application such as addation ,storing the result 
 
-
-	
-	Scenario: Performing Addation operation Using standard Version 
-	#Step1
-	Given : Open  Calculator Application and 
-	And   : Click on View Menu and select the Standard Option
-	Then  : Perform Operations in Standard Mode
-	Then  :  Click on View Menu and select the Scientific Option
-	Then  : Perform Operations in Scientific Mode
-	Then  : Result Should Be 361
+@outline
+    Scenario Outline: Check for result for operation
+	Given  Open the Calculator Application 
+	And   : Select the Mode '<SpecificMode>'
+	Then  : I Enter '<Numb1>' '<numb2>' and perform add operation
```

Comment:
Split this step to several:
I input '...' number
I perform '...' action

Reply:
updated